dask_glm/utils.py

```diff
@@ -5,9 +5,35 @@
 
 import dask.array as da
 import numpy as np
+from functools import wraps
 from multipledispatch import dispatch
 
 
+def normalize():
```

Comment:
I think since we don't use `@normalize` with any arguments, you can remove one level of nestedness here by removing the `def decorator(algo)` function. So like

```python
def normalize(algo):
    @wraps(algo)
    def normalize_inputs(...)
        ...
    return normalize_inputs
```

And then using it like `@normalize` instead of `@normalize()`